tests/api_error_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build apitests
```

Comment:
Not sure why the golang/goland tooling added this. I sure didn't add it manually.

Need to look into it.